template/drush/drushrc.php

```diff
@@ -1,305 +1,6 @@
- * commands when the "--structure-tables-key=common" option is provided.
- * You may add specific tables to the existing array or add a new element.
- */
-# $options['structure-tables']['common'] = array('cache', 'cache_*', 'history', 'search_*', 'sessions', 'watchdog');
-
-/**
- * List of tables to be omitted entirely from SQL dumps made by the 'sql-dump'
- * and 'sql-sync' commands when the "--skip-tables-key=common" option is
- * provided on the command line.  This is useful if your database contains
- * non-Drupal tables used by some other application or during a migration for
- * example.  You may add new tables to the existing array or add a new element.
- */
-# $options['skip-tables']['common'] = array('migration_*');
-
-/**
- * Override specific entries in Drupal's variable system or settings.php (D6/D7 only).
- */
-# $options['variables']['site_name'] = 'My Drupal site';
-# $options['variables']['theme_default'] = 'minnelli';
-# $options['variables']['anonymous'] = 'Visitor';
-
-/**
- * Command-specific execution options:
- *
- * Most execution options can be shared between multiple Drush commands; these
- * are specified as top-level elements of the $options array in the prior
- * examples above.  On the other hand, other options are command-specific, and,
- * in some cases, a shared option needs a different configuration depending on
- * which command is being executing.
- *
- * To define options that are only applicable to certain commands, make an entry
- * in the $command-specific array as shown below.  The name of the command may
- * be either the command's full name or any of the command's aliases.
- *
- * Options defined here will be overridden by options of the same name on the
- * command line.  Unary flags such as "--verbose" are overridden via special
- * "--no-xxx" options (e.g. "--no-verbose").
- *
- * Limitation: If 'verbose' is set in a command-specific option, it must be
- * cleared by '--no-verbose', not '--no-v', and visa-versa.
- */
-
-// Ensure all rsync commands use verbose output.
-# $command_specific['rsync'] = array('verbose' => TRUE);
-
-// Prevent drush ssh command from adding a cd to Drupal root before provided command.
-# $command_specific['ssh'] = array('cd' => FALSE);
-
-// Additional folders to search for scripts.
-// Separate by : (Unix-based systems) or ; (Windows).
-# $command_specific['script']['script-path'] = 'sites/all/scripts:profiles/myprofile/scripts';
-
-// Always show release notes when running pm-update or pm-updatecode.
-# $command_specific['pm-update'] = array('notes' => TRUE);
-# $command_specific['pm-updatecode'] = array('notes' => TRUE);
-
-// Set a predetermined username and password when using site-install.
-# $command_specific['site-install'] = array('account-name' => 'alice', 'account-pass' => 'secret');
-
-// Use Drupal version specific CLI history instead of per site.
-# $command_specific['core-cli'] = array('version-history' => TRUE);
+// Add project-specific code below.
```

Comment:
I'd add a reference to drush's upstream version of this file so users know where to find the removed comments.